Source/AlertControllerView.swift

```diff
@@ -125,18 +130,24 @@ class AlertControllerView: UIView {
     }
 
     private func createTitleLabelConstraints() {
-        addConstraint(NSLayoutConstraint(item: self.titleLabel, attribute: .FirstBaseline, relatedBy: .Equal,
+        let widthOffset = self.visualStyle.contentPadding.left + self.visualStyle.contentPadding.right
+
+        addConstraint(NSLayoutConstraint(item: self.titleLabel, attribute: .Top, relatedBy: .Equal,
```

Comment:
Please keep the individual items' attributes the same (so `.FirstBaseline` and `.Top`) to match `UIAlertController` more closely.